src/ducks/devtools/Queries.jsx

```diff
@@ -0,0 +1,140 @@
+import React, { useState, useMemo } from 'react'
+import sortBy from 'lodash/sortBy'
+import Grid from '@material-ui/core/Grid'
+import Box from '@material-ui/core/Box'
+
+import { useClient } from 'cozy-client'
+import Typography from 'cozy-ui/transpiled/react/Typography'
+import ListItem from 'cozy-ui/transpiled/react/ListItem'
+import ListItemText from 'cozy-ui/transpiled/react/ListItemText'
+import { useSelector } from 'react-redux'
+import format from 'date-fns/format'
+import Inspector from 'react-inspector'
+
+import { NavSecondaryAction, ListGridItem } from './common'
+
+const styles = {
+  panelRight: { height: '100%', overflow: 'scroll', flexGrow: 1 },
+  mono: { fontFamily: 'monospace' }
+}
+
+const FetchStatus = ({ fetchStatus }) => {
+  return (
+    <span
+      className={
+        fetchStatus == 'loaded'
+          ? 'u-valid'
+          : fetchStatus === 'pending'
+          ? 'u-warn'
+          : fetchStatus === 'error'
+          ? 'u-error'
+          : ''
+      }
+    >
+      {fetchStatus}
+    </span>
+  )
+}
+
+const QueryData = ({ data, doctype }) => {
+  const client = useClient()
+
+  const documents = useSelector(state => state.cozy.documents[doctype])
+
+  const storeData = useMemo(() => {
+    return data.map(id => client.hydrateDocument(documents[id]))
+  }, [client, data, documents])
```

Comment:
Not really since documents is the whole `documents` slice (key = doctype, value =documents), whereas in your example `documents` is already an array of documents.
We could do `client.hydrateDocuments(data.map(id => documents[id]))` though but it is not really bringing anything to the table.

Reply:
thankd